apps/web/package.json

```diff
@@ -4,6 +4,7 @@
   "type": "module",
   "scripts": {
     "dev": "svelte-kit dev",
+    "dev-host": "svelte-kit dev --host",
```

Comment:
Please remove this, for people that needs --host can use `pnpm dev --host`, or create a local script